client/landing/gutenboarding/stepper-onboarding/steps.ts

```diff
@@ -0,0 +1,6 @@
+// this a list of all the steps, it's meant to list them, not preserve their order
```

Comment:
I am a bit confused by this comment (particularly the `not preserve their order` part). Which one of the following is true?

- `Steps` is *the list* of the steps in the onboarding flow, which has a predefined order (which follows the property order in the object). But then what does the `not preserve their order` mean here? Also, if this is the *list*, why not use an array which has more explicit ordering between its elements (rather than object)?
- `Steps` is *a set of all possible steps*, which is used to compose the final list (e.g. `[Steps.First, Steps.Third]` like in the original gutenboarding code). But then it seems that `useSteps()` always returns `Object.values(Steps)`, so it looks like it *is* the list. Or is it just a placeholder code?

Either way it looks contradictory (to me), could you clarify? 😄

Reply:
This is obsolete now. Thanks so much for the feedback!